docker/mainnet/connector/connector.mainnet.conf

```diff
@@ -0,0 +1,54 @@
+[Application Options]
+datadir=/root/.connector
+debuglevel=debug
+network=mainnet
+# RPC address to bind
+rpchost=0.0.0.0
+rpcport=9002
+# exchange address
+enginehost=mainnet.exchange.bitlum.io
+engineport=8080
```

Comment:
Do we opened this port only for connector?

Reply:
We do not have our exchange in mainnet and we run with `--disableexchange` option. We can remove this option but i'm not sure if it isn't mandatory or something.